--- pcsc-lite-1.9.9/src/debuglog.c.orig	2022-09-11 13:51:43.000000000 +0200
+++ pcsc-lite-1.9.9/src/debuglog.c	2022-09-12 01:28:32.960862895 +0200
@@ -136,6 +136,8 @@ void log_msg(const int priority, const c
 {
 	char DebugBuffer[DEBUG_BUF_SIZE];
 	va_list argptr;
+	// boolean not to do openlog several times
+	static int syslog_opened = 0;
 
 	if ((priority < LogLevel) /* log priority lower than threshold? */
 		|| (DEBUGLOG_NO_DEBUG == LogMsgType))
@@ -145,6 +147,11 @@ void log_msg(const int priority, const c
 	vsnprintf(DebugBuffer, sizeof DebugBuffer, fmt, argptr);
 	va_end(argptr);
 
+	if (DEBUGLOG_SYSLOG_DEBUG == LogMsgType && !syslog_opened) {
+		openlog(NULL, LOG_PID, LOG_DAEMON);
+		syslog_opened = 1;
+	}
+
 	log_line(priority, DebugBuffer, -1);
 } /* log_msg */
 
@@ -203,7 +210,7 @@ static void log_line(const int priority,
 	unsigned int rv)
 {
 	if (DEBUGLOG_SYSLOG_DEBUG == LogMsgType)
-		syslog(LOG_INFO, "%s", DebugBuffer);
+		syslog(LOG_DEBUG, "%s", DebugBuffer);
 	else
 	{
 		static struct timeval last_time = { 0, 0 };
