--- pcsc-lite-1.1.1/src/Makefile.am.orig	Mon Jul  1 18:28:39 2002
+++ pcsc-lite-1.1.1/src/Makefile.am	Sat Mar 15 16:01:56 2003
@@ -13,7 +13,7 @@
 if PCSC_ARCH_LINUX
 if PCSC_LINUX_USB
 PCSC_MYLDFLAGS      =
-PCSC_INCLUDE_LIBS   = -lfl -ldl
+PCSC_INCLUDE_LIBS   = -lfl -ldl -lpthread
 PCSCD_INCLUDE_LIBS  = -lfl -ldl -lpthread
 PCSC_THREAD_SOURCE  = thread_unix.c
 PCSC_DYNLOAD_SOURCE = dyn_unix.c
@@ -21,7 +21,7 @@
 PCSC_SYSTEM_SOURCE  = sys_unix.c
 else
 PCSC_MYLDFLAGS      =
-PCSC_INCLUDE_LIBS   = -lfl -ldl
+PCSC_INCLUDE_LIBS   = -lfl -ldl -lpthread
 PCSCD_INCLUDE_LIBS  = -lfl -ldl -lpthread
 PCSC_THREAD_SOURCE  = thread_unix.c
 PCSC_DYNLOAD_SOURCE = dyn_unix.c
