--- pcsc-lite-2.5.0/src/winscard.c.orig	2026-05-27 14:39:48.000000000 +0200
+++ pcsc-lite-2.5.0/src/winscard.c	2026-06-07 19:48:38.070569555 +0200
@@ -408,6 +408,12 @@ LONG SCardConnect(/*@unused@*/ SCARDCONT
 			{
 				(void)pthread_mutex_unlock(rContext->mMutex);
 
+				/*
+				 * If it is set to ANY let it do any of the protocols
+				 */
+				if (dwPreferredProtocols & SCARD_PROTOCOL_ANY_OLD)
+					dwPreferredProtocols = SCARD_PROTOCOL_T0 | SCARD_PROTOCOL_T1;
+
 				if (! (dwPreferredProtocols & rContext->readerState.cardProtocol))
 				{
 					rv = SCARD_E_PROTO_MISMATCH;
@@ -708,6 +714,12 @@ LONG SCardReconnect(SCARDHANDLE hCard, D
 			{
 				(void)pthread_mutex_unlock(rContext->mMutex);
 
+				/*
+				 * If it is set to ANY let it do any of the protocols
+				 */
+				if (dwPreferredProtocols & SCARD_PROTOCOL_ANY_OLD)
+					dwPreferredProtocols = SCARD_PROTOCOL_T0 | SCARD_PROTOCOL_T1;
+
 				if (! (dwPreferredProtocols & rContext->readerState.cardProtocol))
 				{
 					rv = SCARD_E_PROTO_MISMATCH;
