--- pcsc-lite-1.1.1/src/Makefile.am.orig	Sat Mar 15 16:06:24 2003
+++ pcsc-lite-1.1.1/src/Makefile.am	Sat Mar 15 16:08:47 2003
@@ -114,7 +114,7 @@
 
 libpcsclite_la_LDFLAGS = $(PCSC_INCLUDE_LIBS) -version-info 0:1:0
 
-pcscd_SOURCES = pcscdaemon.c winscard_svc.c winscard_msg.c \
+pcscd_SOURCES = pcscdaemon.c winscard_svc.c \
 	thread_generic.h readerfactory.h \
 	atrhandler.h configfile.h dyn_generic.h eventhandler.h hotplug.h \
 	ifdhandler.h ifdwrapper.h powermgt_generic.h prothandler.h \
