--- pcsc-lite-1.6.2/src/hotplug_libusb.c.orig	2010-06-30 14:40:58.000000000 +0200
+++ pcsc-lite-1.6.2/src/hotplug_libusb.c	2010-08-08 09:49:14.094819105 +0200
@@ -507,8 +507,8 @@
 		ret = libusb_open(dev, &device);
 		if (ret < 0)
 		{
-			Log2(PCSC_LOG_ERROR, "libusb_open failed: %s",
-				libusb_strerror(ret));
+			Log2(PCSC_LOG_ERROR, "libusb_open failed: %d", ret);
+				/* libusb_strerror(ret)); */
 		}
 		else
 		{
@@ -518,8 +518,8 @@
 
 			if (ret < 0)
 			{
-				Log2(PCSC_LOG_ERROR, "libusb_get_string_descriptor_ascii failed: %s",
-					libusb_strerror(ret));
+				Log2(PCSC_LOG_ERROR, "libusb_get_string_descriptor_ascii failed: %s", ret);
+					/* libusb_strerror(ret)); */
 				readerTracker[i].fullName = strdup(driver->readerName);
 			}
 			else
