diff -Nur pcsc-lite-1.1.1.orig/DRIVERS pcsc-lite-1.1.1/DRIVERS
--- pcsc-lite-1.1.1.orig/DRIVERS	Mon Jul  1 18:28:39 2002
+++ pcsc-lite-1.1.1/DRIVERS	Tue Mar 11 10:52:56 2003
@@ -2,18 +2,18 @@
 
 To install drivers download the driver from http://www.linuxnet.com
 and copy it to a temporary location.  After you have unarchived the files
-copy the directory to /usr/local/pcsc/drivers.  If the directory 
-/usr/local/pcsc/drivers does not exist create it before copying the driver
+copy the directory to /usr/lib/pcsc/drivers.  If the directory 
+/usr/lib/pcsc/drivers does not exist create it before copying the driver
 directory to it.
 
 Example Mac OS X, Linux 2.4:
 
 tar -xzvf driver.bundle-0.1.0.tar.gz
-cp -r driver.bundle /usr/local/pcsc/drivers
-cd /usr/local/pcsc/drivers/driver.bundle
+cp -r driver.bundle /usr/lib/pcsc/drivers
+cd /usr/lib/pcsc/drivers/driver.bundle
 build or use Project Builder to build
 
-The /usr/local/pcsc/drivers directory should contain bundle directories only.
+The /usr/lib/pcsc/drivers directory should contain bundle directories only.
 You must restart pcscd to use the new driver.
 
 
